Documentation/HowToGuides/CreatingAPhysicsButton/README.md

```diff
@@ -0,0 +1,132 @@
+#  Creating A Physics Button
+
+> * Level: Intermediate
+>
+> * Reading Time: 10 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+## Prerequisites
+
+* [Add the Tilia.Interactions.Interactor.Unity] prefab to the scene Hierarchy.
+* [Add the Tilia.Trackers.ColliderFollower.Unity] prefab to the scene Hierarchy.
+* [Install the Tilia.Interactions.Controllables.Unity] package dependency in to your [Unity] project.
+
+## Let's Start
+
+### Step 1
+
+Create a new `Empty` GameObject by selecting `Main Menu -> GameObject -> Create Empty` and rename it to `ButtonContainer`.
+
+![Import Empty](assets/images/ImportEmpty.png)
+
+Expand the `Tilia Interactions Controllables Unity` Package directory in the Unity Project window and select the `Packages -> Tilia Interactions Controllables Unity -> Runtime -> Prefabs -> Physics Joint` directory then drag and drop the `LinearJointDrive` prefab into the Unity hierarchy window so it is a child of the `ButtonContainer` Empty GameObject.
+
+![Drag And Drop LinearJointDrive](assets/images/DragAndDropLinearJointDrive.png)
+
+### Step 2
+
+Select the `Slider -> Interactions.LinearJointDrive -> Internal -> JointContainer -> Joint -> Interactions.Interactable -> MeshContainer -> Cube` GameObject and change the transform properties to:
+
+*  Scale: ` X = 0.2, Y = 0.02, Z = 0.15`
+
+![Change Mesh Scale](assets/images/ChangeMeshScale.png)
+
+### Step 3
+
+Select the `Interactions.LinearJointDrive` GameObject from the Unity Hierarchy and on the `Linear Drive Facade` component set the following properties to:
+
+* Drive Axis: `Y Axis`
+* Start At Initial Target Value: `checked`
+* Initial Target Value: `1`
+* Move To Target Value: `checked`
+* Moved Target Value: `1`
+* Drive Limit: `0.05`
+
+![Configure LinearJointDrive](assets/images/ConfigureLinearJointDrive.png)
+
+### Step 4
+
+Create a new `Empty` GameObject and make it a child of the `ButtonContainer` Empty GameObject, rename it to `ActivationArea`.
+
+Add a `Box Collider` that is going to be the trigger and set the transform properties to:
```

Comment:
It's not setting the transform properties, it's setting the `Box Collider` component properties, plus you need to mention `Is Trigger` is checked.

so:

```
Add a `Box Collider` that is going to be the trigger and set the `Box Collider` component properties to:

* Is Trigger: `checked`
* Size: ` X = 0.25, Y = 0.15, Z = 0.2`
```